src/test/java/core/basesyntax/service/impl/strategy/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.impl.strategy.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.FruitTransactionDto;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.operations.impl.BalanceOperationHandler;
+import core.basesyntax.service.strategy.Operation;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class BalanceOperationHandlerTest {
+    private static final String FRUIT_NAME = "Apple";
+    private static final int BALANCE_QUANTITY = 10;
+    private BalanceOperationHandler balanceOperationHandler;
+    private Storage storage;
+
+    @BeforeEach
+    public void setUp() {
+        storage = new Storage();
+        balanceOperationHandler = new BalanceOperationHandler(storage);
+    }
+
+    @Test
+    public void apply_AddToFruitStorage_ok() {
+        FruitTransactionDto dto = new FruitTransactionDto(Operation.BALANCE, FRUIT_NAME,
```

Comment:
we can have lots of DTO classes across our application, but a `transaction` is more clear and unique name for it
```suggestion
        FruitTransactionDto transaction = new FruitTransactionDto(Operation.BALANCE, FRUIT_NAME,
```